trio/_core/tests/test_kqueue.py

```diff
@@ -0,0 +1,25 @@
+import select
+import subprocess
+
+import pytest
+
+from ... import _core
+
+use_kqueue = hasattr(select, 'kqueue')
+pytestmark = pytest.mark.skipif(not use_kqueue, reason="kqueue platforms only")
+
+
+async def test_kqueue_monitor_proc_event():
+    p = subprocess.Popen(['sleep', '0.1'])
```

Comment:
So all that stuff about zombies means you don't need this process to sleep. Maybe `true` and `false` and then you can also check the exit code?